inductiva/tasks/__init__.py

```diff
@@ -1,4 +1,4 @@
 #pylint: disable=missing-module-docstring
 from inductiva.tasks.task import Task
 from inductiva.tasks.run_simulation import run_simulation
-from inductiva.tasks.methods import get, list  # pylint: disable=redefined-builtin
+from inductiva.tasks.methods import get, to_dict  # pylint: disable=redefined-builtin
```

Comment:
I think we no longer need the pylint directive.

Reply:
Done